client/nsxt_client.go

```diff
@@ -293,3 +293,46 @@ func (c *nsxtClient) GetLoadBalancerStatus(loadBalancerID string) (loadbalancer.
 	loadBalancerStatus, _, err := c.apiClient.ServicesApi.ReadLoadBalancerServiceStatus(c.apiClient.Context, loadBalancerID, nil)
 	return loadBalancerStatus, err
 }
+
+func (c *nsxtClient) ListAllSections() ([]manager.FirewallSection, error) {
```

Comment:
Is it better to name it ListAllFirewallSections?

Reply:
fixed in 8aef830